perun-wui-core/src/main/java/cz/metacentrum/perun/wui/json/managers/RegistrarManager.java

```diff
@@ -236,7 +236,7 @@ public static Request createApplication(Application application, List<Applicatio
 		if (data != null) {
 			client.put("data", data);
 		}
-		return client.call(REGISTRAR_MANAGER + "createApplication");
+		return client.call(REGISTRAR_MANAGER + "submitApplication");
```

Comment:
If its not temporary change, we should create own method for ``submitApplication`` or rename java method too. Also for now I'm missing server-side change (pull-request).

Reply:
The method was added in https://github.com/CESNET/perun/pull/2812, which was now merged to master.